app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    copy_command, source_file, destination_file = command.split(" ")
+
+    if copy_command == "cp":
+        if source_file == destination_file:
+            return
+
+        with (open(source_file, "r") as file_in,
+              open(destination_file, "w") as file_out):
+            for line in file_in:
+                file_out.write(line)
```

Comment:
Can be written more simply and without iterating in "for" loop for all lines. Like a:
`file_out.write(file_in.read())`